hal/halx86/xbox.cmake

```diff
@@ -21,6 +21,7 @@ list(APPEND HAL_XBOX_SOURCE
     generic/usage.c
     generic/bios.c
     generic/portio.c
+    generic/x86bios.c
```

Comment:
Can't say for PC-98, but on Xbox there's simply no VESA BIOS. One day we could want to inject it to make proprietary Nvidia driver happy though, but then Pentium III has no problems with V86 mode.

Reply:
If we want to avoid building x86Bios on those platforms, then it could be made a stub there (and maybe even Vista+ videoprt would load with stubs, falling back to V86 if it's available there).